site/layouts/partials/stylesheet.html

```diff
@@ -1,11 +1,11 @@
 {{ if hugo.IsProduction -}}
 {{ if eq .Page.Params.direction "rtl" -}}
-<link href="/docs/{{ .Site.Params.docs_version }}/dist/css/ouds-web.rtl.min.css" rel="stylesheet" {{ printf "integrity=%q" .Site.Params.cdn.css_rtl_hash | safeHTMLAttr }}>
```

Comment:
Note for future selves: we decided not to handle the `integrity` here in order to simplify the entire process, and to avoid creating new "hash" entries in `hugo.yml`.